pkg/ebpf/config.go

```diff
@@ -96,7 +96,7 @@ func NewDefaultConfig() *Config {
 		UDPConnTimeout:               30 * time.Second,
 		TCPConnTimeout:               2 * time.Minute,
 		MaxTrackedConnections:        65536,
-		ConntrackMaxStateSize:        65536,
+		ConntrackMaxStateSize:        131072, // MaxTrackedConnections * 2
```

Comment:
I think the default value that actually matters is the one in the pkg/process.
Also, can we raise the short lived connections buffer size too?

Reply:
Both done. I think we can get rid of `ebpf.NewDefaultConfig()` entirely, but will wait until the network/ebpf refactor happens.